src/target/source/codegen_c_host.h

```diff
@@ -42,7 +43,12 @@ class CodeGenCHost : public CodeGenC {
 
   void InitGlobalContext();
   void AddFunction(const PrimFunc& f);
-
+  /*!
+   * \brief Add functions from the (unordered) range to the current module in a deterministic order.
```

Comment:
nit: maybe mention that it's "in a deterministic order to aid debugging."

Reply:
done